### PR TITLE
added-delete

### PR DESCRIPTION
**Pull request by mlyoung25**

### CODE REVIEW

File: pkg/cmd/delete.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+Copyright © 2022 Zeet, Inc - All Rights Reserved
+test
```
Comment (zzh8829) [inline pkg/cmd/delete.go:3]: remove the word "test"

File: pkg/api/repo.go [hunk 2]
```diff
@@ -8,7 +8,19 @@ import (
 )
 
 type Repo struct {
-	ID uuid.UUID `copier:"Id"`
+	Branch string    `json:"branch"`
+	ID     uuid.UUID `copier:"Id"`
+}
+
+func (c *Client) DeleteRepo(ctx context.Context, repoID uuid.UUID, branch string) error {
```
Comment (zzh8829) [inline pkg/api/repo.go:15]: the branch variable is not needed anymore

**Review by zzh8829 — CHANGES_REQUESTED** [review]

**Comment by mlyoung25** [comment]
idk what to do but i fixed the requests

**Comment by zzh8829** [comment]
👍 nice

**Pull request merged**